creations/src/components/Clients.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import '../App.css';
-import Figure from 'react-bootstrap/Figure';
-import img1 from './images/img 1.png';
+import Figure from 'react-bootstrap/Figure'
+import img1 from '../images/img1.png';
```

Comment:
./ is used to get something within on under same folder
but images were 1 folder above it
So use ../ instead to go one folder up and then fetch image